client/my-sites/posts/post-controls/index.jsx

```diff
@@ -167,25 +174,6 @@ const getAvailableControls = props => {
 	return controls;
 };
 
-const getControlElements = controls => controls.map( ( control, index ) =>
-	<li
-		className={ classNames( { 'post-controls__disabled': control.disabled } ) }
-		key={ index }
-	>
-		<a
-			className={ `post-controls__${ control.className }` }
-			href={ control.href }
-			onClick={ control.disabled ? noop : control.onClick }
-			target={ control.target ? control.target : null }
-		>
-			<Gridicon icon={ control.icon } size={ 18 } />
-			<span>
-				{ control.text }
-			</span>
-		</a>
-	</li>
-);
-
 export const PostControls = props => {
```

Comment:
The more logic there involves the viewport size, all the checks for each button (user capabilities, post status, even feature flags) and even a CSS animation that relies on having 2 separate container elements.
I can't think of a way of moving controls around with all these conditions, except the one already in place.